interfaces/builtin/basedeclaration_test.go

```diff
@@ -466,6 +468,7 @@ func (s *baseDeclSuite) TestConnection(c *C) {
 		"fwupd":            true,
 		"location-control": true,
 		"location-observe": true,
+		"lxd":              true,
 		"mir":              true,
 		"modem-manager":    true,
 		"udisks2":          true,
```

Comment:
Please add another test to TestSlotInstallation() for lxd, underneath the docker one.